### PR TITLE
Br aws accumulo

### PR DESCRIPTION
**Pull request by kthare10**
Cloudformation and scripts needed to spawn Accumulo cluster in AWS

### CODE REVIEW

File: aws-accumulo/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+See official documentation for more information.
+
+## How to use this cloudformation?
+### Pre-requisites
+1. User must have AWS account with privilges to create/delete IAMRole, IAMPolicy and IAMProfile
+2. Key pair has been created
+### Create a Accumulo Stack
+Create a stack on AWS Cloudformation service by using accumuloCloudFormation.json. 
+#### Logon to AWS Console and Search for Cloudformation service
+![Cloudformation](../master/aws-accumulo/images/aws1.png)
```
Comment (mjstealey) [inline aws-accumulo/README.md]: Image references should not be branch based, but rather repository hierarchy based (keeps images as files in the repository).
Alternatively images can be added to an issue and then use the reference URL provided by Github (images are not part of the repository but are references to Github's internal storage).

**Comment by kthare10** [comment]
@mjstealey Updated the Readme as suggested

**Review by mjstealey — APPROVED** [review]
+1 - Images will now be shown regardless of branch name

**Comment by mjstealey** [comment]
@kthare10 - May want to document which environment variables should be updated by the user prior to deployment such that a password isn't just the default `secret`.

Otherwise 👍

Look forward to seeing this demo'd

**Pull request merged**